account-service/src/main/java/com/bdonor/accountservice/Controller/AccountController.java

```diff
@@ -1,76 +1,111 @@
+            //System.out.println(url.toString());
+            HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+            conn.setRequestMethod("GET");
+            conn.setRequestProperty("Accept", "application/String");
+
+            if (conn.getResponseCode() != 200) {
+                throw new RuntimeException("Failed : HTTP error code : " + conn.getResponseCode());
+            }
 
-        String json = new Gson().toJson(Service_functions.getAll());
+            String output = "E";
+            BufferedReader br = new BufferedReader(new InputStreamReader((conn.getInputStream())));
 
-        return json;
+            System.out.println("Output from Server .... \n");
+            while ((output = br.readLine()) != null) {
+                URLink = output;
+                System.out.println(URLink);
+                LatLong = URLink.split(",");
+            }
+
+            conn.disconnect();
+
+        } catch (MalformedURLException e) {
+
+            e.printStackTrace();
+
+        } catch (IOException e) {
+
+            e.printStackTrace();
+        }
+        mapper.save(new User(bloodGroup, firstName, lastName, email, bCryptPasswordEncoder.encode(password), addressline, postcode, LatLong[0], LatLong[1]));
+        return "User added to Dynamo Database";
+    }
+    
+    @GetMapping(value = "/getUser/{firstName}/{email}")
+    public ResponseEntity<User> getUserDetails(@PathVariable String firstName, @PathVariable String email) { // Working
+        User user = dynamoRepo.getSingleUser(firstName, email);
+        System.out.println(user.toString());
+        return new ResponseEntity<User>(user, HttpStatus.OK);
     }
 
-    @ResponseBody
-    @RequestMapping("/updateUser/{bloodGroup}/{firstname}/{surname}/{email}/{password}/{addressline}/{password}") // Works Partially - Creates new user instead of updating current
-    public String updateUser( @PathVariable String bloodGroup , @PathVariable  String firstname, @PathVariable  String surname, @PathVariable  String email, @PathVariable  String password, @PathVariable  String addressline, @PathVariable  String postcode ){
-        User Update = Service_functions.Update(bloodGroup, firstname,  surname,  email,  password,  addressline,  postcode);
-        return Update.toString();
+    @PutMapping(value = "/updateUser")
+    public void updateUserDetails(@RequestBody User user) { // Working -- Not sure how i will re-encrypt new users password
+        dynamoRepo.updateUserDetails(user);
     }
 
-    @ResponseBody
-    @GetMapping("/deleteUser/{firstname}") // Works
-    public String deleteUser(@PathVariable String firstname ){
-        Service_functions.deleteByfirstName(firstname);
-        return "Deleted" + firstname;
+    @DeleteMapping(value = "/delete/{firstName}/{email}")
+    public void deleteUserDetails(@PathVariable String firstName, @PathVariable String email) { // Working
+        User user = new User();
```

Comment:
https://github.com/we-3hackathon/bdonor/blob/e81f5b4d18581c56ab3e031675033d7a09cb62ef/account-service/src/main/java/com/bdonor/accountservice/Controller/AccountController.java#L95-L98

Reply:
PL35, A1 (Don't worry about this one, just a code for later addressing)